apps/sms/js/threads.js

```diff
@@ -18,6 +18,10 @@
     this.messages = new Map();
   }
 
+  Thread.prototype.getDraft = function() {
+    return this.isDraft ? Drafts.get(this.id) : Drafts.byThreadId(this.id);
```

Comment:
in my best dreams, I want a "Draft" object that has the same interface that a "Thread" object. So we could call "delete" on it directly without even doing a check like this.